drivers/spi/Kconfig.stm32

```diff
@@ -17,10 +17,16 @@ config SPI_STM32_INTERRUPT
 	help
 	  Enable Interrupt support for the SPI Driver of STM32 family.
 
+config SPI_STM32_DMA
+	bool "STM32 MCU SPI DMA Support"
+	help
+	  Enable DMA support for the SPI Driver of STM32 family.
+
```

Comment:
Please add some usage guidance such that: when enabled, SPI DMA mode is used for SPI instances that enable dma channels their device tree node.

Reply:
done